src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java

```diff
@@ -194,4 +194,8 @@ public void verifyMessageAboutNewPassword(String userName) {
 			true
 		);
 	}
+
+	public void verifyFBButtonVisible() {
```

Comment:
@bekcunning, actually that won't be a problem as long as each subclass defines it's own ```facebookSignUpButton``` instance variable (which you've done). 

So, for example, when you call ```verifyFBButtonVisible()``` on the SpecialUserLoginPageObject [here](https://github.com/Wikia/selenium-tests/blob/UC-190-1/src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java#L47), the method inside of base class is called [here](https://github.com/Wikia/selenium-tests/blob/UC-190-1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java#L1219-L1221), but it'll check if the subclass has a ```facebookSignUpButton``` defined which it does [here](https://github.com/Wikia/selenium-tests/blob/UC-190-1/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/login/SpecialUserLoginPageObject.java#L39-L40). So it'll use the proper selector "#mw-content-text .wikia-button-facebook".

When you call ```verifyFBButtonVisible()``` on the DropDownComponentObject [here](https://github.com/Wikia/selenium-tests/blob/UC-190-1/src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java#L29) it'll use [this selector](https://github.com/Wikia/selenium-tests/blob/UC-190-1/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java#L40-L41) (after you've renamed it to  ```formConnectWithFbButton```).

If the base class doesn't have a ```formConnectWithFbButton``` instance variable defined (like the SpecialNewFilesPageObject class), that's when it'll fall back to the base class version which is what's happening forced login test.